src/testshade/simplerend.cpp

```diff
@@ -84,6 +85,23 @@ struct MicrofacetParams {
 struct DebugParams {
     ustring tag;
 };
+struct ParameterTestParams {
+    int int_param;
+    float float_param;
+    Color3 color_param;
+    Vec3 vector_param;
+    ustring string_param;
+    int int_array[3];
```

Comment:
Wondering if a size other than 3 can be used to avoid any confusion with triples.  By confusion I mean anyone debugging llvm IR might confuse triple values with an array length 3.  And a number other than 3 might stick out better.  maybe [5]